.mvn/wrapper/maven-wrapper.properties

```diff
@@ -16,4 +16,4 @@
 # under the License.
 wrapperVersion=3.3.2
 distributionType=only-script
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.8/apache-maven-3.8.8-bin.zip
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.9/apache-maven-3.9.9-bin.zip
```

Comment:
wasn't there was a maven restriction to use GluonFX ?

Reply:
There was, but the GluonFX plugin 1.0.26-SNAPSHOT doesn't have it anymore.